src/pages/Groups/Group1/Screen1.tsx

```diff
@@ -1,7 +1,90 @@
 import React from 'react'
+import LeftOutlined from '@ant-design/icons/lib/icons/LeftOutlined'
+import UnorderedListOutlined from '@ant-design/icons/lib/icons/UnorderedListOutlined'
+import {
+  SucessfullyDeletedEvent,
+  TopNav,
+  BotNav,
+  Header,
+  StyledButton,
+  StyledTick,
+  DeletedEventText,
+  EventName,
+  EventTime,
+  UndoIcon,
+  OptionsBoxes,
+  GreyBoxDelete,
+  GreyBoxSort,
+  GreyBoxText,
+  StyledGreybox,
+  Booking,
+  BookingLocation,
+  BookingCCA,
+  BookingTiming,
+  StyledEdit,
+  StyledDustbin,
+  StyledCCAIcon,
+  CentreNav,
+} from './styles/Screen1.styled'
+import homeIconUnselected from '../../../assets/Group1/homeIconUnselected.svg'
+import facilitiesIconSelected from '../../../assets/Group1/facilitiesIconSelected.svg'
+import calenderIconUnselected from '../../../assets/Group1/calenderIconUnselected.svg'
+import profileIconUnselected from '../../../assets/Group1/profileIconUnselected.svg'
+import tickIconRH from '../../../assets/Group1/tickIconRH.svg'
+import dustbinIconBlack from '../../../assets/Group1/dustbinIconBlack.svg'
+import Schedule from '../../../assets/Group1/Schedule.svg'
+import greendustbin from '../../../assets/Group1/dustbinIconGreen.svg'
+import greenedit from '../../../assets/Group1/editIconGreen.svg'
+import bandroom from '../../../assets/Group1/bandroomIcon.svg'
 
 const Screen1 = () => {
-  return <>this is screen 1</>
+  return (
+    <>
+      <div>
```

Comment:
Unnecessary to wrap everything in a div again since <> in react does the same thing. Doesn't do any harm though.